### DISCUSSION TITLE
Aqara Radiator Thermostat E1 (SRTS-A01)  with external Sensor but without HA

### DISCUSSION THREAD

**Discussion by aifrog** [Koenkk/zigbee2mqtt#19357] [Devices]
Did anyone make this work without HA?

THx, H-C

**Comment by RealPete** [comment]
If you are looking for a solution with openHAB, here is my rule:
```
configuration: {}
triggers:
  - id: "1"
    configuration:
      itemName: Temperatur_Sensor_Schlafzimmer_MQTT_Sensor
    type: core.ItemStateUpdateTrigger
conditions: []
actions:
  - inputs: {}
    id: "2"
    configuration:
      type: application/javascript
      script: actions.get("mqtt",
        "mqtt:broker:283b588d8b").publishMQTT("zigbee2mqtt/Heizung Aqara
        Schlafzimmer/set/sensor_temp",
        items.getItem('Temperatur_Sensor_Schlafzimmer_MQTT_Sensor').state);
    type: script.ScriptAction
```

Of course you have to change the mqtt broker id and item names to match with your system (and the mqtt topic).
And manually set the sensor to "external" in the zigbee2mqtt frontend.

If you are looking for a method with zigbee2mqtt only: I had no luck with that. I couldn't bind them together.
